### ISSUE TITLE
The download-progress not fires on AppImage

### ISSUE THREAD

**Issue by dlliu114** [electron-userland/electron-builder#6749]
* **Electron-Builder Version**: 22.10.5

* **Node Version**: 12.22.4

* **Electron Version**: 13.6.3
* **Electron Type (current, beta, nightly)**:

* **Target**:  Ubuntu 18.04 AppImage

I'm implementing all the events (update-available, update-downloaded and download-progress, etc) to make the update process more friendly to application.

I've made a progress bar which is working great on Windows with NSIS and on MacOS with DMG, but the event download-progress is never fires on AppImage.  It's a bug on electron-builder?

**Comment by mmaietta** [comment]
Please try the latest version of electron-builder and electron-updater.
AFAICT, the code is identical for emitting that event
https://github.com/electron-userland/electron-builder/search?l=TypeScript&q=DOWNLOAD_PROGRESS

**Comment by xxxace** [comment]
@mmaietta is the download speed too fast that the reason why the download-progress don't fires?

**Comment by Kerinlin** [comment]
@xxxace Yes, i have tested,when the package is so small,the download-progress event didn't emit

**Comment by mmaietta** [comment]
Just took a look at the code. The callback `onProgress` only fires at 1sec intervals after a 1sec delay
https://github.com/electron-userland/electron-builder/blob/d42fa79a8bdacd81565bfab37db7f97e49a5a185/packages/builder-util-runtime/src/ProgressCallbackTransform.ts#L17
https://github.com/electron-userland/electron-builder/blob/d42fa79a8bdacd81565bfab37db7f97e49a5a185/packages/builder-util-runtime/src/ProgressCallbackTransform.ts#L32-L34
